samples/ControlCatalog/Pages/ColorPickerPage.xaml

```diff
@@ -0,0 +1,29 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:cpp="clr-namespace:Avalonia.Controls.Primitives;assembly=Avalonia.Controls.ColorPicker"
```

Comment:
@maxkatz6 I think something similar is required for DataGrid; however, is there any way to remove this namespace?

Reply:
Same as below